tools/build_pytorch_libs.bat

```diff
@@ -194,11 +201,12 @@ goto:eof
   : which is said to become an error in the future.
   : As an alternative, we should use forward slashes instead.
   : Here those paths should be escaped before passing to CMake.
-  set NVTOOLEXT_HOME=%NVTOOLEXT_HOME:\=/%
-  set CUDNN_INCLUDE_DIR=%CUDNN_INCLUDE_DIR:\=/%
-  set CUDNN_LIB_DIR=%CUDNN_LIB_DIR:\=/%
-  set CUDNN_LIBRARY=%CUDNN_LIBRARY:\=/%
-  set PYTORCH_PYTHON_LIBRARY=%PYTORCH_PYTHON_LIBRARY:\=/%
+  if not "%NVTOOLEXT_HOME%" == "" set NVTOOLEXT_HOME=%NVTOOLEXT_HOME:\=/%
```

Comment:
Why do you need the extra empty tests here?

Reply:
This is because when the variable is empty, the variable will be set to the command itself rather than remain empty. Here is an example:
``` cmd
$>IF DEFINED NVTOOLEXT_HOME (echo NVTOOLEXT_HOME is defined) else (echo NVTOOLEXT_HOME is not defined)
NVTOOLEXT_HOME is not defined

$>echo %NVTOOLEXT_HOME%
%NVTOOLEXT_HOME%

$>set NVTOOLEXT_HOME=%NVTOOLEXT_HOME:\=/%

$>echo %NVTOOLEXT_HOME%
%NVTOOLEXT_HOME:\=/%
```